### PR TITLE
lizard augment egalitarianism

### PR DESCRIPTION
**Pull request by Aticius**
Finally, they can go see the Biomechanical Engineers we fired.

Unlikely to break anything unless i'm criminally bad at spelling.

image

### CODE REVIEW

**Comment by Aticius** [comment]
!review

File: html/changelogs/aticius-lizardaugments.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+author: ChangeMe
```
Comment (Arrow768) [inline html/changelogs/aticius-lizardaugments.yml:2]: You need to change that.

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Afaik the unauthorised augment restrictions were quite intentional due to the difference between their and other species tech levels.

I'll discuss that with the lore masters, until then it's a dem

**Comment by Aticius** [comment]
Changes addressed because my head is full of water

**Review by Arrow768 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**